app/views/layouts/application.html.slim

```diff
@@ -24,14 +24,12 @@ html
 
     .govuk-width-container
       = govuk_phase_banner(tag: { text: "Beta" }) do
-        | This is a new service, your
-        =< govuk_link_to("feedback", "#")
-        |  will help us improve it.
+        = t('phase_banner', feedback_url: govuk_link_to("feedback", ENV.fetch('FEEDBACK_URL', '#feedback_url-to-come'))).html_safe
```

Comment:
You'll need to test this, but I think if a locale key ends `_html` the system assumes its raw HTML content and you won't need to run `html_safe` on it. So I'd suggest changing the key to `phase_banner_html` and removing the `html_safe` here and see if that works. See: https://makandracards.com/makandra/52455-rails-including-html-in-your-i18n-locales

Reply:
done